build/windows/omsupply_server.suf

```diff
@@ -4404,7 +4409,7 @@ g_HandleSystemReboot();
 </SessionVar>
 <SessionVar>
 <Name>%omsupply_db_name%</Name>
-<Value>omsupply-database</Value>
+<Value>open-msupply</Value>

```

Comment:
not part of the bug fix, just tidying up.

Reply:
I have no idea what it's doing :) but can you rename the database name? e.g. what if it is just a version upgrade?